book/website/community-handbook/translation/translation-localisation.md

```diff
@@ -0,0 +1,70 @@
+(ch-translation-localisation)=
+
+## Localisation Platforms and Translation
+
+
+Localisation (l10n) and internationalisation (i18n) are important aspects in the design of any open-source project or documents.
+Internationalisation allows open-source project to support and satisfy the needs of multiple locales, thus “enabling” localisation, which is the adaptation of it to meet the language, cultural of a specific target locale.
+The World Wide Web Consortium (W3C) defines internationalisation (i18n) as:
+> "The design and development of a product, application or document content that enables easy localisation for target audiences that vary in culture, region, or language"
+
+
+ **A Translation Management System (TMS)** manages the localisation process from the beginning of a translation process until the finished product.
+TMS are widely used in Open source projects because they offer many advantages such as workflow, automation, transparency, and fast project delivery.
+
+Localisation involves more than just translation, which only transforms text. Localisation addresses other factors such as text length and cultural references.  **A Translation Management System (TMS)** manages the localisation process from the beginning of a translation process until the finished product. They are widely used in Open source projects because they offer many advantages such as workflow, automation, transparency, and fast project delivery.
```

Comment:
This is repeated text from the paragraph above.

Reply:
should i remove the duplicate at line 16, or i should remove the one at line 12

Reply:
please keep the second